File: app/src/main/java/com/guuguo/learnsave/view/IMainView.java
package com.guuguo.learnsave.view;

import com.guuguo.learnsave.bean.entity.GankBean;

import java.util.List;

/**
 * 主界面的接口
 * Created by panl on 15/12/22.
 */
public interface IMainView extends IBaseView {
    void showProgress();
    void hideProgress();
    void showNoMoreData();
    void showErrorView();
    void showMeiziList(List<GankBean> lMeiziList);
}

File: app/src/main/java/com/guuguo/learnsave/view/IMainView.kt
package com.guuguo.learnsave.view

import com.guuguo.learnsave.model.entity.GankModel

/**
 * 主界面的接口
 * Created by panl on 15/12/22.
 */
interface IMainView : IBaseView {

    fun showMeiziList(lMeiziList: List<GankModel>)
}
